lambda_function.py

```diff
@@ -0,0 +1,80 @@
+import json
+import urllib.parse
+import boto3
+import logging
+#from botocore.exceptions import ClientErro
+
+print('Loading function') 
+region = 'us-east-1'
+#bucket_name = 'csv-bucket-3534r35d'
+csv_bucket_name = 'bucket-for-converted-csv-objects'
+csv_object_name = 'converted-csv-object'
+
+
+recieved_object_name = ""
+response = ""
+s3 = boto3.client('s3')
+s3_client = boto3.client('s3')
+
+def lambda_handler(event, context):
+    #print("Received event: " + json.dumps(event, indent=2))
+
+    buildCSV = ""
+    # Get the object from the event and show its content type
+    bucket = event['Records'][0]['s3']['bucket']['name']
+    key = urllib.parse.unquote_plus(event['Records'][0]['s3']['object']['key'], encoding='utf-8')
+    recieved_object_name = event['Records'][0]['s3']['object']['key']
+    
+    csv_object_name = "csv-object-of-" + recieved_object_name
+    
+    #print(csv_object_name)
+    
+    create_bucket(csv_bucket_name)
+    #print(event['Records'][0]['s3']['object'])
+    #print(event['Records'][0]['s3']['object']['key'])
+    
+    #print(type(recieved_object_name))
+    
+    try:
+        response = s3.get_object(Bucket=bucket, Key=key)
+        print("CONTENT TYPE: " + response['ContentType'])
+        #print(response)
+        #print(response['Body'])
+        body = json.loads(response['Body'].read().decode("utf-8"))
+        for v,k in body.items():
+            buildCSV += str(v) + "," + str(k)
+            buildCSV += "\n"
```

Comment:
I think this is not a proper logic for CSV conversion
If this is the input 
{
 "key1": "val1",
 "key2": "val2"
}
will produce a result like this
key1,val1
key2,val2
which is not a proper format